CHANGELOG

```diff
@@ -1,3 +1,14 @@
+snowplow-ecommerce 0.9.1 (2025-05-12)
+---------------------------------------
+## Summary
+This release fixes a bug when using BigQuery V2 type columns
+
+## Features
```

Comment:
NIT: Here (and under Fixes if any) we include the commit name as is and the details go to the summary, if needed.